src/Utils.ts

```diff
@@ -167,7 +168,19 @@ const Utils = {
             cqPath = normalizePath(cqPath);
         }
         return cqPath;
-    }
+    },
+
+    fetchModelOnBackNav(componentProps: ComponentProps): void {
```

Comment:
`Utils.ts` may not be the right location for that function. WDYT? Any suggestion?